code/game/machinery/vending.dm

```diff
@@ -962,21 +963,22 @@
 
 /obj/machinery/vending/wallmed1
 	name = "\improper NanoMed"
-	desc = "A wall-mounted version of the NanoMed."
+	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilisation and much more."
```

Comment:
Why?

Reply:
so it's more explanatory what the contents inside are.